master/buildbot/test/unit/steps/test_source_git.py

```diff
@@ -309,7 +316,10 @@ def test_mode_full_clean_ssh_key_1_7(self):
         self.expectOutcome(result=SUCCESS)
         self.expectProperty(
             'got_revision', 'f6ad368298bd941e934a41f3babc827b2aa95a1d', self.sourceName)
-        return self.runStep()
+        yield self.runStep()
+
+        expected = b'#!/bin/sh\nssh -i "%s" "$@"\n' % (unicode2bytes(ssh_key_path),)
```

Comment:
Can we use `.format()` for string formatting?

Reply:
Changed to using `.format()`!